include/openssl/e_os2.h

```diff
@@ -350,6 +350,16 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+# ifdef __STDC_VERSION__
+#  if (__STDC_VERSION__ >= 201112L)
+#   include <stdnoreturn.h>
+#  endif
```

Comment:
You're right - we've opened the box of pandora or in this case, the box of compiler feature detection macros...